src/tests/unit/tests/DetailsView/components/report-export-component.test.tsx

```diff
@@ -70,31 +77,32 @@ describe('ReportExportComponent', () => {
 
     test('render with dialog closed', () => {
         props.isOpen = false;
-        const wrapper = shallow(<ReportExportComponent {...props} />);
-        expect(wrapper.debug()).toMatchSnapshot();
+        const renderResult = render(<ReportExportComponent {...props} />);
+        expect(renderResult.asFragment()).toMatchSnapshot();
     });
 
     test('render with dialog open', () => {
-        const wrapper = shallow(<ReportExportComponent {...props} />);
-        expect(wrapper.debug()).toMatchSnapshot();
+        const renderResult = render(<ReportExportComponent {...props} />);
+        expect(renderResult.asFragment()).toMatchSnapshot();
     });
 
     test('dismiss dialog', () => {
         dismissDialogMock.setup(d => d()).verifiable(Times.once());
 
-        const wrapper = shallow(<ReportExportComponent {...props} />);
-        const exportDialog = wrapper.find(ExportDialog);
-        exportDialog.props().onClose();
+        render(<ReportExportComponent {...props} />);
+
+        const exportDialog = getMockComponentClassPropsForCall(ExportDialog);
+        exportDialog.onClose();
```

Comment:
you can replace these 2 lines into oneline
getMockComponentClassPropsForCall(ExportDialog).onClose();
test once again